src/main/java/org/codehaus/mojo/license/download/LicensedArtifactResolver.java

```diff
@@ -41,12 +41,10 @@
 import org.apache.maven.project.ProjectBuilder;
 import org.apache.maven.project.ProjectBuildingException;
 import org.apache.maven.project.ProjectBuildingRequest;
-import org.codehaus.mojo.license.api.ArtifactFilters;
-import org.codehaus.mojo.license.api.DefaultThirdPartyTool;
-import org.codehaus.mojo.license.api.MavenProjectDependenciesConfigurator;
-import org.codehaus.mojo.license.api.ResolvedProjectDependencies;
+import org.codehaus.mojo.license.api.*;
```

Comment:
please don't use asterisks for imports

Reply:
Done for my code.

In the existing code, there were already some asterisk imports.